File: payment-manager-spa/src/app/_models/web-store.ts
export class WebStore {
    id: string;
    successUrl: string;
    errorUrl: string;
    failureUrl: string;
    url: string;
    singleMerchantStore: boolean;
    storeName: string;

    constructor(id: string, successUrl: string, errorUrl: string, failureUrl: string, url: string, singleMerchantStore: boolean, storeName: string){
        this.errorUrl = errorUrl;
        this.failureUrl = failureUrl;
        this.singleMerchantStore = singleMerchantStore;
        this.successUrl = successUrl;
        this.url = url;
        this.id = id;
        this.storeName = storeName;
    }
}

File: PaymentManager.Api/Data/Entities/WebStore.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace PaymentManager.Api.Data.Entities
{
    public class WebStore
    {
        public Guid Id { get; set; }
        public string SuccessUrl { get; set; }
        public string ErrorUrl { get; set; }
        public string FailureUrl { get; set; }
        public string Url { get; set; }
        public bool SingleMerchantStore { get; set; }
        public ICollection<WebStorePaymentService> PaymentOptions { get; set; }
        public ICollection<Merchant> Merchants { get; set; }
        [Timestamp]
        public byte[] TableVersion { get; set; }
    }
}